client/image_tag.go

```diff
@@ -1,21 +1,23 @@
 package client
 
 import (
-	"errors"
```

Comment:
It's the standard format that `goimport` uses https://godoc.org/golang.org/x/tools/cmd/goimports